src/test/js/log-karaoke/noStages.js

```diff
@@ -67,5 +69,19 @@ module.exports = {
                 })
             ;
         });
-    }
+    },
+
+    'Check whether a log which exceed 150kb contains a link to full log and if clicked it disappear': function (browser) {
+        const blueRunDetailPage = browser.page.bluePipelineRunDetail().forRun('noStages', 'jenkins', 1);
+        // first turn on xpath to get the nodes we want
+        browser.useXpath();
+        var xPath = '//div[@class="logConsole"][14]';
```

Comment:
Ok, something magic about row number 14? Still ... I think this could/should be encapsulated inside the relevant page object as a function, hiding all the XPath junk away from the actual test.